imaging.rst

```diff
@@ -183,4 +183,4 @@ QR Code® is a registered trademark of DENSO WAVE INCORPORATED.
     :Example Print logo 2 from dotlines 0 to 861:
         .. code-block:: none
 
-            write('\x1b\xfa\x02\x00\x00\x03\x5e')   #  862 dotlines total
```

Comment:
I don't see a diff here, might a whitespace thing. Did you happen to render this and verify that it looks okay?